platform-api/src/routers/forms.py

```
@@ -0,0 +1,164 @@
+    import json
+    import time
+
+    key = f"rate:forms:{composite_key}"
+    raw = await kv.get(key)
+    now = int(time.time())
+
+    if raw is None:
+        # First write: initialize count and timestamp
+        data = {"count": 1, "started_at": now}
+        await kv.put(key, json.dumps(data), expirationTtl=RATE_LIMIT_WINDOW_SECS)
+        return 1 > RATE_LIMIT_MAX
```

Ruff (0.15.11)

[warning] Yoda condition detected

Rewrite as `RATE_LIMIT_MAX < 1`

(SIM300)